### ISSUE TITLE
Plots HTML layout

### ISSUE THREAD

**Issue by dberenbaum** [iterative/dvc#6603]
With support for more plots and images that could be scattered around the project, having some level of organization and grouping of plots in the HTML would be helpful, such as:

* Organizing by directory structure or filename
* Organizing by stages in dvc.yaml
* Organizing by plot type
* Organizing by plot title
* If https://github.com/iterative/dvc/discussions/5980 is implemented, organizing by name/label/key

It would be useful to discuss how plots are being organized for other products like Studio and VSCode.

**Comment by dberenbaum** [comment]
Also consider how this impacts https://github.com/iterative/dvclive/issues/82 and https://github.com/iterative/dvclive/issues/150.

**Comment by dberenbaum** [comment]
@shcheklein Will the HTML layout in DVC impact Studio or VSCode layouts?

**Comment by shcheklein** [comment]
@dberenbaum no, only from the product perspective if we decide to take some ideas out of it.

**Comment by dberenbaum** [comment]
More likely it will be the reverse 😄
